pytest.ini

```diff
@@ -1,5 +1,8 @@
 [pytest]
+# For fedora
```

Comment:
This really feels like a hack to me. IMHO we remove this and specific that for development setup you need to make sure pg_config is in your PATH. The pytest fixture uses pg_config to determine the path of binaries such as pg_ctl. 

Then we can also remove the env = line below.

Reply:
Agreed, it's hacky.
pg_config does not come with the postgres apt package by default, so we should probably mention that somewhere too.

Reply:
Agree. But I would apply this change in a new PR.

Reply:
Agree. But I would apply this change in a new PR.